### ISSUE TITLE
Support for importing PNGs

### ISSUE THREAD

**Issue by mathcampbell** [Andrew-Dickinson/photonic-etcher#7]
it be possible to implement a PNG importer? I know most people are usiong this to do PCB etching but I'd like to etch actual bitmaps onto a piece of metal and a striaght PNG improter would be helpful.
Thanks

**Comment by Andrew-Dickinson** [comment]
Can you explain a bit more about your use case? One concern I have with bitmaps is that the Photon screens are mostly monochrome, which means if you want to etch non-binary pixel data you'll need to do it using exposure time. That is definitely possible using the file formats we write to, but it's not easy to do with how we've set things up. I don't think greyscale support is something we would ever introduce into this project, since it has very little relevance for PCB manufacturing

**Comment by digiexchris** [comment]
I have a use-case, etching something like this onto an aluminum plate. It's monochrome already. Making reproduction machinery name plates. I'd love to use my photon as a generic photolithography device.
![image](https://github.com/Andrew-Dickinson/photonic-etcher/assets/5871642/df8110f4-eaff-4258-9c6c-3ede5c0d1532)

**Comment by Andrew-Dickinson** [comment]
Yep that sort of use case does seem like a good thing for us to support. I don't have a ton of bandwidth for this project these days, but I'll leave this issue open to track the feature request

**Comment by mathcampbell** [comment]
Arrhhh sorry I never saw the first reply to this!

My use case is actually etching brass & silver for jewellery (but also
signage, so same as the other reply!)

I have found a way to “hack” it using an online generator but it’s pretty
hacky and hasn’t always worked. A specific “png to photon” tool would be
stellar

Math
